source/jormungandr/jormungandr/realtime_schedule/sytral.py

```diff
@@ -91,7 +110,7 @@ def _call(self, url):
             extra={'rt_system_id': unicode(self.rt_system_id)},
         )
         try:
-            return self.breaker.call(requests.get, url, timeout=self.timeout)
```

Comment:
Shouldn't you keep the timeout (maybe in parameter build) ?

Reply:
I added both with the same value. Into param struct or directly within the pybreaker param.
I don't know what is better. Any idea ?

Reply:
It seens to be more confortable to separate data parameters and pure technical parameters like time out. 
It is done for others realtime connector.